### PR TITLE
feat: Added support for Access Only Collaboration 

### PR DESCRIPTION
**Pull request by antusus**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -2817,9 +2817,9 @@
       }
     },
     "caniuse-lite": {
-      "version": "1.0.30001252",
-      "resolved": "https://registry.npmjs.org/caniuse-lite/-/caniuse-lite-1.0.30001252.tgz",
-      "integrity": "sha512-I56jhWDGMtdILQORdusxBOH+Nl/KgQSdDmpJezYddnAkVOmnoU8zwjTV9xAjMIYxr0iPreEAVylCGcmHCjfaOw==",
+      "version": "1.0.30001399",
```
Comment (lukaszsocha2) [inline package-lock.json:2820]: do we want to commit that ?

File: src/schemas/collaboration.ts [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+	 */
+	expires_at?: string;
+	/**
+	 * The email address used to invite an unregistered collaborator, if they are not a registered user.
+	 */
+	invite_email?: string;
+	/**
+	 * The file or folder to which access is granted. The field is null when the collaboration status is pending.
+	 */
+	item?: any;
```
Comment (lukaszsocha2) [inline src/schemas/collaboration.ts:100]: item?: File | Folder | WebLink ?

File: tests/integration_test/__tests__/collaborations.test.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+		await context.user.dispose();
+		await context.collaborator.dispose();
+		context.folder = null;
+		context.user = null;
+		context.collaboratorEmail = null;
+	});
+
+	it('creates and removes collaboration', async() => {
+		let collaboration;
+		collaboration = await context.client.collaborations.create(
```
Comment (lukaszsocha2) [inline tests/integration_test/__tests__/collaborations.test.js:28]: any specific reason why this is not inlined?

Comment (lukaszsocha2) [reply tests/integration_test/__tests__/collaborations.test.js:28]: `let collaboration = await context.client.collaborations.create(`

**Comment by lukaszsocha2** [comment]
Two questions left. I pushed a commit with proposed changes. Tell me if you like them.

File: src/schemas/collaboration.ts (hunk 2 above)
Comment (antusus) [reply src/schemas/collaboration.ts:100]: We do not have those mappings :(

File: package-lock.json (hunk 1 above)
Comment (antusus) [reply package-lock.json:2820]: So the npm is complaining we are using outdated database and if you fix it it bumps `caniuse` which is fine

**Review by lukaszsocha2 — APPROVED** [review]

**Pull request merged**